spec/requests/api/v1/statuses_spec.rb

```diff
@@ -26,16 +26,20 @@
       end
 
       context 'when post includes filtered terms' do
```

Comment:
This is likely not the most ideal way to test this, but it was one of the tests I could find for custom filters, there didn't actually seem to be tests directly for custom filters.